packages/manager/core/order/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@ovh-ux/manager-core-order",
+  "version": "0.1.0",
+  "private": true,
+  "description": "OVHcloud manager core order.",
```

Comment:
```suggestion
  "description": "OVHcloud Manager express order URL generator.",
```

Reply:
It's not dedicated to express order URL generation